<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplePlugin</class>
 <widget class="QDockWidget" name="SamplePlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ROVI Project Controls</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Utility functions for getting images and 3D clouds</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_random">
        <property name="text">
         <string>Move Object [Random]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_im">
        <property name="text">
         <string>Get Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_scan">
        <property name="text">
         <string>Get Scan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Controls to run implemented pose estimation and motion planning</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_m2">
        <property name="text">
         <string>Pose Estimation [M2]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_showPose">
        <property name="text">
         <string>Visualize Pose Estimation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_run">
        <property name="text">
         <string>Calculate Path  [Linear interpolation]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_rrt">
        <property name="text">
         <string>Calculate Path [RRT]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn1">
        <property name="text">
         <string>Run Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_solve">
        <property name="text">
         <string>Run Combined Vision &amp;&amp; Robotics Solution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="_label">
        <property name="text">
         <string>ROVI Project 2019 
 Emil Vincent Ancker &amp; Mikkel Larsen</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Condensed</family>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>The plugin is based on the handed plugin example</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
